tutorials/tts/FastPitch_ChineseTTS_Training.ipynb

```diff
@@ -0,0 +1,3456 @@
+      "com_SF_ce1597.wav  com_SF_ce2212.wav  com_SF_ce362.wav\t com_SF_ce980.wav\r\n",
+      "com_SF_ce1598.wav  com_SF_ce2213.wav  com_SF_ce363.wav\t com_SF_ce981.wav\r\n",
+      "com_SF_ce1599.wav  com_SF_ce2214.wav  com_SF_ce364.wav\t com_SF_ce982.wav\r\n",
+      "com_SF_ce16.wav    com_SF_ce2215.wav  com_SF_ce365.wav\t com_SF_ce983.wav\r\n",
+      "com_SF_ce160.wav   com_SF_ce2216.wav  com_SF_ce366.wav\t com_SF_ce984.wav\r\n",
+      "com_SF_ce1600.wav  com_SF_ce2217.wav  com_SF_ce367.wav\t com_SF_ce985.wav\r\n",
+      "com_SF_ce1601.wav  com_SF_ce2218.wav  com_SF_ce368.wav\t com_SF_ce986.wav\r\n",
+      "com_SF_ce1602.wav  com_SF_ce2219.wav  com_SF_ce369.wav\t com_SF_ce987.wav\r\n",
+      "com_SF_ce1603.wav  com_SF_ce222.wav   com_SF_ce37.wav\t com_SF_ce988.wav\r\n",
+      "com_SF_ce1604.wav  com_SF_ce2220.wav  com_SF_ce370.wav\t com_SF_ce989.wav\r\n",
+      "com_SF_ce1605.wav  com_SF_ce2221.wav  com_SF_ce371.wav\t com_SF_ce99.wav\r\n",
+      "com_SF_ce1606.wav  com_SF_ce2222.wav  com_SF_ce372.wav\t com_SF_ce990.wav\r\n",
+      "com_SF_ce1607.wav  com_SF_ce2223.wav  com_SF_ce373.wav\t com_SF_ce991.wav\r\n",
+      "com_SF_ce1608.wav  com_SF_ce2224.wav  com_SF_ce374.wav\t com_SF_ce992.wav\r\n",
+      "com_SF_ce1609.wav  com_SF_ce2225.wav  com_SF_ce375.wav\t com_SF_ce993.wav\r\n",
+      "com_SF_ce161.wav   com_SF_ce2226.wav  com_SF_ce376.wav\t com_SF_ce994.wav\r\n",
+      "com_SF_ce1610.wav  com_SF_ce2227.wav  com_SF_ce377.wav\t com_SF_ce995.wav\r\n",
+      "com_SF_ce1611.wav  com_SF_ce2228.wav  com_SF_ce378.wav\t com_SF_ce996.wav\r\n",
+      "com_SF_ce1612.wav  com_SF_ce2229.wav  com_SF_ce379.wav\t com_SF_ce997.wav\r\n",
+      "com_SF_ce1613.wav  com_SF_ce223.wav   com_SF_ce38.wav\t com_SF_ce998.wav\r\n",
+      "com_SF_ce1614.wav  com_SF_ce2230.wav  com_SF_ce380.wav\t com_SF_ce999.wav\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "# DataChineseTTS directory looks like\n",
+    "!ls NeMoChineseTTS/DataChinese -R"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c4f33db9",
+   "metadata": {},
+   "source": [
+    "## 2. Creating manifests \n",
+    "\n",
+    "We've created `scripts/dataset_processing/tts/sfbilingual/get_data.py` script that reads the `DataChineseTTS/SF_bilingual/text_SF.txt` provided with the dataset and generates the following fields per each datapoint:\n",
+    "1. `audio_filepath`: location of the wav file\n",
+    "2. `duration`: duration of the wav file\n",
+    "3. `text`: original text supplied by SF dataset\n",
+    "    \n",
+    "After that, the script would splits the datapoints into 3 buckets `train_manifest.json`, `val_manifest.json` and `test_manifest.json`, respectively."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "eb063cf4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!(cd NeMoChineseTTS && \\\n",
+    "    python get_data.py \\\n",
+    "        --data-root ./DataChinese/sf_bilingual_speech_zh_en_vv1/SF_bilingual/\n",
```

Comment:
need to add a `\` at the end of Line 922, otherwise, indent error happens.

``` bash
!(cd NeMoChineseTTS && \
    python get_data.py \
        --data-root ./DataChinese/sf_bilingual_speech_zh_en_vv1/SF_bilingual/ \
        --manifests-path ./)
```